projects/kit/styles/components/progress-bar.less

```diff
@@ -104,6 +104,18 @@
         border-radius: inherit;
     }
 
+    &[tuiProgressFixedGradient] {
+        .progressValue({
+            border-radius: inherit;
+
+            width: 100% !important;
+            margin-right: calc(-100% + var(--tui-progress-percent));
+
+            clip-path: inset(0 calc(100% - var(--tui-progress-percent)) 0 0 round var(--tui-radius-m));
+            transition: all var(--tui-duration) linear;
```

Comment:
Why `all` and not only `clip-path` ?

Btw, Firefox does not contains any animation – it is known issue. Setting `all` does not enable this animation